### ISSUE TITLE
Low tetrahydrocortisol (THF) plus 5-alpha-THF/tetrahydrocortisone (THE) ratio 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7724]
**Preferred term label:**
Low tetrahydrocortisol (THF) plus 5-alpha-THF/tetrahydrocortisone (THE) ratio 

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
614662 
PMID:21325058  2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000185
- label: Low tetrahydrocortisol (THF) plus 5-alpha-THF/tetrahydrocortisone (THE) ratio
- parents: [Abnormality of circulating cortisol level (HP:0011731)](https://hpo.jax.org/app/browse/term/HP:0011731)
- pmid: PMID:21325058
- def: The relative concentration of tetrahydrocortisol plus 5-alpha-tetrahydrocortisol relative to that of tetrahydrocortisone is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7724
- synonyms:
